Pipfile

```diff
@@ -6,21 +6,21 @@ name = "pypi"
 [packages]
 anyio = "==3.7.1"
 asgiref = "==3.7.2"
-click = "==8.1.3"
+click = "==8.1.7"
```

Comment:
Does this package need to be listed here? I checked because `click` is used for building CLI applications and I found it is a dependency of `uvicorn` so it will be installed anyway, albeit with a looser version restriction (`>= 7.0`). The only reason to list it here would be to ensure the latest version is used. Is this required?

(This is obviously not a blocking issue)

Reply:
yeah, maybe it's not required. i'lll create another pr removing it